crates/api_common/src/site.rs

```diff
@@ -320,6 +322,10 @@ pub struct GetSiteResponse {
   pub my_user: Option<MyUserInfo>,
   pub all_languages: Vec<Language>,
   pub discussion_languages: Vec<LanguageId>,
+  /// deprecated, use field `tagline` or /api/v3/tagline/list
```

Comment:
You might be able to use a rust deprecated tag also. Not sure if its worth it though, or what it gives us for an API.

`#[deprecated(since="0.5.0", note="please use `new_method` instead")]`